Formula/zydis.rb

```diff
@@ -0,0 +1,24 @@
+class Zydis < Formula
+  desc "Fast and lightweight x86/x86-64 disassembler library"
+  homepage "https://zydis.re"
+  url "https://github.com/zyantific/zydis/archive/v2.0.2.tar.gz"
+  sha256 "bd711102a5a30096562a7cb60bafbc9c4a2441ce5463a59f4d16f2dd73f9fb72"
+  head "https://github.com/zyantific/zydis.git"
+
+  depends_on "cmake" => :build
+
+  def install
+    mkdir "zydis-build" do
+      system "cmake", "..", *std_cmake_args
+      system "make", "install"
+      bin.install "ZydisDisasm"
+      bin.install "ZydisInfo"
+      bin.install "ZydisFuzzIn"
+      bin.install "ZydisPerfTest"
```

Comment:
Why are these required? If upstream does not typically install those files as part of `make install`, we probably don't want to ship them.

Reply:
Removed ZydisFuzzIn and ZydisPerfTest. ZydisDisasm and ZydisInfo are apparently both not installed, but useful.

Reply:
Would you please open an issue upstream, asking why these tools are not installed and whether they are considered standard / should be installed by default?

Reply:
@athre0z just fixed the issue, it was not meant to not install them. He will make a new release (2.0.3). I will push the formula as soon as he is finished.

Reply:
fixed!